README.md

```diff
@@ -56,6 +56,23 @@ jhack model rm
 
 Happy hacking!
 
+
+# Disclaimer
+
+Many of the commands jhack offers are pretty destructive or bring a high risk of catastrophic failure and random explosions.
+
+These commands will prompt the user for confirmation before proceeding, after, whenever possible, showing the low-level (Juju) commands it would run.
+
+## Enabling devmode
```

Comment:
Perhaps a sentence to explain that the intent for devmode is to disable the prompts? I think it is implicit below - from the name devmode could also be something intended for use during development for jhack (that would have been my initial guess). E.g.:

```
In devmode, destructive commands will be run without requiring a confirmation prompt.
```